docs/source/configuration/settings-reference.md

```diff
@@ -9,3 +9,7 @@ This is a summary of all the configuration options and what they control.
 
 Navigation levels depth used in the navigation endpoint calls. Increasing this is useful
 for implementing fat navigation menus. Defaults to `1`.
+
+# defaultBlockType
+
+By default this is `'text'` but can be set to other block type string from addons.
```

Comment:
Please reword:

The default block type in Volto is "text", which uses the current draftjs-based implementation of rich text editor. It can be set to other block type implementations.